src/components/Person/Person.jsx

```diff
@@ -1 +1,51 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner
+    = (sex === 'm'
+      ? 'wife'
+      : 'husband');
```

Comment:
```suggestion
  const partner = sex === 'm'
      ? 'wife'
      : 'husband';
```
I think it can be written like this

Reply:
agree, fix your formatting